src/modules/shared/components/Overlay/Overlay.module.scss

```diff
@@ -9,7 +9,7 @@ $overlay-opacity-light: 0.4;
 	pointer-events: none;
 	opacity: 0;
 	position: fixed;
-	top: 0;
+	top: -1px;
```

Comment:
wouldn't it be cleaner to change the size of the navigation items to 10rem instead of 9.9rem?
```
@media screen and (min-width: 1400px)
.Navigation_c-navigation__item__ZRI0T {
    margin-bottom: 0.1rem;
    position: relative;
    height: 9.9rem; // change to 10 rem
}
```

Reply:
⬆️, negative margins are always a "risk" so if it's possible to fix it with this I'd chose that option.